brvs-core/src/main/java/iroha/validation/service/impl/ValidationServiceImpl.java

```diff
@@ -45,11 +47,11 @@ public void verifyTransactions() {
           final String hex = ValidationUtils.hexHash(transaction);
           logger.info("Got transaction to validate: " + hex);
           for (Validator validator : validators) {
-            if (!validator.validate(transaction)) {
-              final String canonicalName = validator.getClass().getCanonicalName();
-              transactionSigner.rejectAndSend(transaction, canonicalName);
-              logger.info(
-                  "Transaction has been rejected by the service. Failed validator: " + canonicalName);
+            final ValidationResult validationResult = validator.validate(transaction);
+            if (!validationResult.getStatus().equals(Verdict.VALIDATED)) {
```

Comment:
This may be simplified. Enumerations in Java can be compared using `==`.  I'd rather write `Verdict.VALIDATED!=validationResult.getStatus()`